### DISCUSSION TITLE
Least Upper Bound

### DISCUSSION THREAD

**Discussion by HLe4s-kaist** [prosyslab-classroom/cs524-program-analysis#430] [Art competition]
![LeastUpperBound](https://github.com/prosyslab-classroom/cs524-program-analysis/assets/161300429/5a7d8709-1725-4b60-9027-c2e6d84dc6ed)
[English follows]
## Name
강찬우
## Tool You Used
[Adobe Fire Fly](https://firefly.adobe.com/upload/inpaint)
## Short Description
지구가 담겨있는 유리 구슬이 존재하며, 그 밖에 우주인이 한 명 보인다. 그런데 그 밖에 또 다른 유리벽이 존재한다. 
 * 그림을 그린지 오래되어서, 프롬프트가 기억이 안납니다. 죄송합니다. 

## Long Description (up to 3 sentences)
Least Upper Bound라는 개념은, 넘으려고 해도 절대 넘을 수 없는 벽과 같은 것으로 표현하면 좋을거 같다고 생각했습니다. 그래서 매우 큰 물체인 지구조차 하나의 유리 구슬에 담겨있을 정도로 벽이 있고, 그 벽을 넘어선 존재라도 다시 새로운 넘을 수 없는 벽에 갇힌 존재라는 것을 표현하는 것으로 해당 개념을 나타내어 봤습니다. 여기서 재밌는 부분은 우주인의 LUB와 지구의 LUB가 다르다는 것입니다. 

[Eng]
## Name
ChanU Kang
## Tool You Used
[Adobe Fire Fly](https://firefly.adobe.com/upload/inpaint)
## Short Description
A glass sphere contains the Earth, and an astronaut is visible outside of it. Beyond that, there is another glass wall.
* It's been a long time since I drew this, so I can't remember the exact prompt. I apologize.

## Long Description (up to 3 sentences)
I thought it would be good to represent the concept of the LUB as a wall that cannot be surpassed, no matter how hard one tries. Thus, I depicted even a big thing like the Earth confined within a glass sphere, and an astronaut who has surpassed that boundary only to find themselves enclosed by another wall. The interesting part here is that the LUB for the astronaut is different from the LUB for the Earth.

**Comment by KihongHeo** [comment]
Interesting. Nice.
